features/submitting_references.feature

```diff
@@ -17,3 +17,16 @@ Feature: references
     Then the new application choice status is "awaiting_references"
     When "s.skinner@springfield-elementary.edu" provides a reference
     Then the new application choice status is "application_complete"
+
+  Scenario: an application is sent to a provider after it's received two references and 5 working days elapse after submission
+    Given an application choice has "unsubmitted" status
+    And the candidate has specified "j.moriarty@uni.ac.uk" and 's.skinner@springfield-elementary.edu' as referees
+    And the date is "2019-11-04"
+    And the candidate takes action "submit"
+    And "j.moriarty@uni.ac.uk" provides a reference
+    And "s.skinner@springfield-elementary.edu" provides a reference
+    Then the new application choice status is "application_complete"
+    When the date is "2019-11-11"
```

Comment:
Is it worth adding a line in here like `And the daily application cron has run`, to trigger it? Not sure how granular to be in the description